### ISSUE TITLE
Logs page shows (incorrectly) the same tag added multiple times

### ISSUE THREAD

**Issue by AlesyaBreenkova** [TaxoPress/TaxoPress#1335]
I created a post. Added tags to it from Dandelion, while setting "Auto Terms Limit" to "0" under the "Options" tab in "Manage Auto Terms". Everything worked fine.

Went back to the settings to set "Auto Terms Limit" to "1". Then I quick edited my post a few times. The logs showed that the same term was added each time (please see the screenshot). 
I don't know why that happened, my guess it's because the tag contains multiple words separated with a colon and commas. 

![Logs  Tag added multiple times_LI](https://user-images.githubusercontent.com/31319775/159285409-cd96e9ee-3dff-420a-abd4-3687b90649d6.jpg)

Also, apparently there are 3 different tags which are very similar - "Superintelligence", "Superintelligence: Paths", "Superintelligence: Paths, Dangers, Strategies" (see screenshots below) and the system at some point fails to recognize them as being different. 

That said, the post page looks fine, tags are added only once, as designed, while the logs page shows the same tag is added each time.

![Tag link highlighted](https://user-images.githubusercontent.com/31319775/159296292-b8891e49-a1bd-4f10-8fe0-4fbe3cfa3859.png)

![Tag link highlighted (Superintelligence Paths)](https://user-images.githubusercontent.com/31319775/159296320-0bd720be-1659-480e-823f-a7b1eed54d29.png)

![Tag link highlighted (Superintelligence Paths, Dangers, Strategies )](https://user-images.githubusercontent.com/31319775/159296359-410ed614-4aac-4b3a-af97-e32878a1d1fb.png)

I need to note, I've chosen to go with longer posts on purpose, to see how well the plugin works on longer content.  Of course, not every user of WordPress and Auto Terms is going to encounter bugs like this.

**Comment by ojopaul** [comment]
Okay, this is as a result of the 'save_post' filter running more than once and for every time the filter run, a log entry is made.

I'll check if there's any reasonable way we can make each log unique but it's good to know that doesn't change the 'save_post' from running more than once and every action attached to it will always process for all the time it run.

This is only noticeable because an entry was made.

**Comment by ojopaul** [comment]
@stevejburge can you check if you're able to recreate this issue?

**Comment by stevejburge** [comment]
@ojopaul Yes, but it seem to connect to terms with commas. Let's leave this one for now
